zeppelin-web/src/components/websocketEvents/websocketMsg.service.js

```diff
@@ -180,10 +180,11 @@ angular.module('zeppelinWebApp').service('websocketMsgSrv', function($rootScope,
       });
     },
 
-    getEditorSetting: function(replName) {
+    getEditorSetting: function(orderId, replName) {
       websocketEvents.sendNewEvent({
         op: 'EDITOR_SETTING',
         data: {
+          orderId: orderId,
```

Comment:
@minahlee Thank you for your review.
i will check your opinion part and make fix it.